.circleci/config.yml

```diff
@@ -202,16 +203,23 @@ workflows:
       #      - js
 
       # PERF TESTING
-      - perf_tests:
+      - master_perf_tests:
+          filters:
+            tags:
+              only: /^(?!canary).*$/
+
```

Comment:
shouldn't the perf require previous steps? At least the build?

Reply:
I don't think that the performance tests require the build files. The performance tests still run, and if they don't require those files then we can start the performance tests right away to help save some time.
https://app.circleci.com/pipelines/github/godaddy-wordpress/coblocks/7706/workflows/121983b9-74eb-43e1-8174-40a9bf3c8655

Reply:
Oh okay, right, I thought the perf files required to have a build done.